whois-update/src/main/java/net/ripe/db/whois/update/mail/MailGatewaySmtp.java

```diff
@@ -31,18 +35,28 @@ public class MailGatewaySmtp implements MailGateway {
     private final LoggerContext loggerContext;
     private final MailConfiguration mailConfiguration;
     private final JavaMailSender mailSender;
+    private final UndeliverableMailDao undeliverableMailDao;
+    private final OutgoingMessageDao outgoingMessageDao;
 
-    @Value("${mail.smtp.enabled}")
+    @Value("${mail.app.path}")
```

Comment:
Rename to something like `web.baseurl` to be consistent with the other properties, and set to the base web app URL for that environment, e.g. `https://rc.db.ripe.net/db-web-ui/`